matrix.c

```diff
@@ -76,11 +76,6 @@ Matrix_t matrix_mult_t(Matrix_t a, Matrix_t b)
   return result;
 }
 
-bool matrix_mult_valid(Matrix_t a, Matrix_t b)
```

Comment:
I think this function is important so our program do not break because of invalid input matrices.